basic-frontend/styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.activeZone {
+  width: 1000px;
+  position: absolute;
+  left: 50%;
+  margin-left: -500px;
+}
+
+.title {
+  color: white;
+  font-family: "Verdana";
+  margin-top: 0.6em;
+  margin-left: 0.1em;
+  font-size: 2em;
+}
+
+.BGtitle {
+  background-color: rgb(255, 90, 31);
+  height: 5em;
+}
+
+.BGtabs {
```

Comment:
I've never seen css classes starting with capital letters. Did you have a reason for that? I wouldn't consider it good practice

Reply:
Just to be easier to find the background classes, I didn't know it's bad practice. I will decapitalize them.